internal/action/actions.go

```diff
@@ -160,6 +160,53 @@ func (h *BufPane) Center() bool {
 	return true
 }
 
+// CursorToViewTop moves the cursor to the top of the view,
+// offset by scrollmargin unless at the beginning or end of the file
+func (h *BufPane) CursorToViewTop() bool {
+	v := h.GetView()
+	h.Buf.ClearCursors()
+	scrollmargin := int(h.Buf.Settings["scrollmargin"].(float64))
+	bStart := display.SLoc{0, 0}
+	if h.Diff(v.StartLine, bStart) == 0 {
+		scrollmargin = 0
+	}
+	h.Cursor.GotoLoc(h.LocFromVLoc(display.VLoc{
+		SLoc:    h.Scroll(v.StartLine, scrollmargin),
+		VisualX: 0,
+	}))
+	return true
+}
+
+// CursorToViewCenter moves the cursor to the center of the view
+func (h *BufPane) CursorToViewCenter() bool {
+	v := h.GetView()
+	h.Buf.ClearCursors()
+	h.Cursor.GotoLoc(h.LocFromVLoc(display.VLoc{
+		SLoc:    h.Scroll(v.StartLine, h.BufView().Height/2),
+		VisualX: 0,
+	}))
+	return true
+}
+
+// CursorToViewBottom moves the cursor to the bottom of the view,
+// offset by scrollmargin unless at the beginning or end of the file
+func (h *BufPane) CursorToViewBottom() bool {
+	v := h.GetView()
+	h.Buf.ClearCursors()
+	scrollmargin := int(h.Buf.Settings["scrollmargin"].(float64))
+	height := h.BufView().Height
+	bEnd := h.SLocFromLoc(h.Buf.End())
+	lastLine := h.Scroll(v.StartLine, height-1)
```

Comment:
mega-nit: `height` is used just once in this function, so you could use `h.BufView().Height` directly instead.

Reply:
Fixed.